### PR TITLE
Fix/profile image fallback color

### PR DESCRIPTION
**Pull request by osmaczko**
required by: https://github.com/status-im/status-desktop/pull/5424

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [x] is this a breaking change?
    - [x] use the dedicated `BREAKING CHANGE` commit message section
    - [x] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [x] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

**Review by alexandraB99 — APPROVED** [review]
lgtm

**Review by noeliaSD — APPROVED** [review]
Nice!

File: src/StatusQ/Core/Theme/StatusLightTheme.qml [hunk 1]
```diff
@@ -145,15 +145,19 @@ ThemePalette {
...
 
-    accountColors: [
-        getColor('blue'),
-        getColor('brown2'),
-        getColor('violet'),
-        "#1D806F",
-        getColor('red2'),
-        getColor('green2'),
-        getColor('purple'),
-        getColor('brown')
```
Comment (PascalPrecht) [inline src/StatusQ/Core/Theme/StatusLightTheme.qml:156]: Why are we removing `accountColors`?
Those are needed for wallet account options.

**Review by PascalPrecht — CHANGES_REQUESTED** [review]

File: src/StatusQ/Core/Theme/StatusLightTheme.qml (hunk 1 above)
Comment (osmaczko) [reply src/StatusQ/Core/Theme/StatusLightTheme.qml:156]: Yes, I've changed all occurrences of `accountColors` to `userCustomizationColors` from: https://www.figma.com/file/WMmmNowe0sTVCaxadUB8mE/Desktop-Foundations?node-id=1%3A609

"total number of colours in the user customisation colour set is 12, but in some contexts only a subset of these colours will be used e.g. 10 colours, or 8 colours from that 12 colour pallet, they can be used and referenced throughout the product wherever user customisation colours are required e.g. this set of colours is used for colouring Accounts in the user's wallet, and is also used in the Community create and edit screens."     John

**Review by PascalPrecht — APPROVED** [review]

File: src/StatusQ/Core/Theme/StatusLightTheme.qml (hunk 1 above)
Comment (osmaczko) [reply src/StatusQ/Core/Theme/StatusLightTheme.qml:156]: Sure, done.

**Pull request merged**